backend/views/emails/org_invite.html

```diff
@@ -7,7 +7,10 @@
 </p>
 
 <p>
-Please click <a href="<%= config.domain %>/#/activate/<%= company.realm %>/<%= token %>">here</a> to activate your account. 
+Please click <a href="<%= config.indabaDomain %>/activate/<%= company.realm %>/<%= token %>">here</a> to activate your account.
```

Comment:
I guess the client and backend were on the same server before? Should we change this in all the message html files now or in a separate ticket?